terraform/modules/route53-logging/main.tf

```diff
@@ -0,0 +1,15 @@
+
+
+resource "aws_route53_resolver_query_log_config" "dns_logs" {
+  name            = "dns_logs"
+  destination_arn = aws_cloudwatch_log_group.aws_route53_logs_com.arn
+
+  tags = {
```

Comment:
`tags` map is empty and will need to be supplied with something. Perhaps a variable?

Reply:
I was going to add the ones I put in the variable list. I'll have a think